pages/Tables.js

```diff
@@ -53,80 +34,231 @@ export default () => {
+            </li>
+          </ul>
+          <p>
+            Note: Text wrapping applies by default on both views and it wraps
+            the text in different lines on a row.
+          </p>
         </div>
+
         <div>
-          <ol>
+          <h5>Anatomy</h5>
+          <p>
+            A table is composed by it’s basic table elements. If users need to
+            interact with the table, additional elements can be added.
+          </p>
+          <b>Table elements:</b>
+          <ul>
             <li>
-              <h4>Container</h4>
-              Container holds all data table content.
+              Column header - Column headers are titles for the column content.
             </li>
             <li>
-              <h4>Column title</h4>
-              if necessary a table can contain a title.
+              Table content - Table content displays all the raw data. As
+              default tables use 25 rows per page.
             </li>
             <li>
-              <h4>Column header</h4>
-              Column headers are titles for the column content.
+              Table pagination - Table pagination is a simple navigation method
+              that lets you split a huge amount of content while indicating that
+              more pages exist.
             </li>
+          </ul>
+
+          <b>Interactive elements:</b>
+          <ul>
             <li>
-              <h4>Sorting tool</h4>
-              Sorting tools allow users to reorder the table content.
+              <b>Table title</b> - Used to differentiate between multiple
+              elements or multiple tables on the same page.
             </li>
             <li>
-              <h4>Table content</h4>
-              Table contains raw data.
+              <b>Badges</b> - Badges are coloured pills to display statuses on a
+              table.
             </li>
             <li>
-              <h4>Row checkbox</h4>
-              To allow users to select a row and perform an action.
+              <b>Editable tables</b> - Editable tables allow the user to
+              manipulate data on a different page. It is great for the entry of
+              tabular data like in the case of a spreadsheet. To enter edit mode
+              provide an Edit button on the top of the page. Clicking Save or
+              Cancel will bring you back to the view mode.
             </li>
-          </ol>
-        </div>
-      </section>
-      <section>
-        <h3>Fixed tables</h3>
-        <div>
-          Fixed tables should contain a width bar to be able to scroll through
-          the right once the user clicks in the container
+          </ul>
         </div>
-        <img style={{ width: '100%' }} src={FixedTable} />
-      </section>
-      <section>
-        <h3>Sorting and filtering</h3>
         <div>
-          Table related actions like sorting and filtering should be kept on the
-          table container as a header.
+          <h5>Mobile</h5>
```

Comment:
This component is not quite big, would be good to split into smaller pieces, maybe one for each section